workspace/ti.cc23xx/ti.mcu.cc23xx/OneShotGpt3.em.zig

```diff
@@ -34,14 +34,22 @@ pub const EM__TARG = struct {
     }
 
     pub fn enable(msecs: u32, handler: HandlerFxn, arg: em.ptr_t) void {
+        start(msecs * 1000, handler, arg);
```

Comment:
For consisitency, I think this should be ustart (because it takes microseconds as an arg).